### DISCUSSION TITLE
Admin UI: Allow Modification of Product Create Form

### DISCUSSION THREAD

**Discussion by jamescjensen** [medusajs/medusa#6405] [Feature Requests]
Neither the product creation form (packages/admin_ui/ui/src/domain/products/new/index.tsx) nor the editing form (packages/admin_ui/ui/src/domain/products/edit/index.tsx) are configurable when making plugins. New fields added to the product entity (or new related entities) cannot be managed on these forms.

[Widgets can be injected into the top or bottom of the product details page](https://docs.medusajs.com/admin/widgets) which makes the editing side of things slightly less of a problem, but nothing is available during product creation.

Additionally, there's a use case for completely replacing an existing field input on the form: if I want the product description to be editable with more of a WYSIWYG-style editor, there's no way to do that right now - I can only create a separate widget and stick it at the bottom of the page, there's no way to replace the existing basic textarea.

**Comment by TufisStefan** [comment]
Are there any updates regarding this or any reasonable workarounds? We really need the option to extend both products and product categories and we must support create/update from the Admin UI, any suggestions would be appreciated.
